### PR TITLE
Style Guide and Diataxis Chrysalis Review - Part 1

### PR DESCRIPTION
**Pull request by charlesthompson3**
Reviewed and updated content based on Tech Writer Style Guide and Diataxis for: 

- Chrysalis Intro
- Firefly Verification of Downloads
- Node Software

(Semi-test run to see if this PR remains stable; further review incoming but if you want to take a swing at it, @iotaledger/tech-writers, go for it)

### CODE REVIEW

File: docs/firefly/verify_download.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Verify your Firefly download on Windows, maxOS and/or Linux.
+description: Verify your Firefly download on Windows, maxOS, and/or Linux.
```
Comment (aleksei-korolev) [inline docs/firefly/verify_download.md:2]: Pick one, and or or.

File: docs/firefly/verify_download.md [hunk 2]
```diff
@@ -8,13 +8,14 @@ keywords:
 - windows
 - macOS
 - linux
+- how to
 ---
 # Verify your Firefly Desktop Download
 
-When you download Firefly Desktop, you may want to verify its authenticity to make sure that you downloaded the correct one from the IOTA Foundation GitHub repository. In this tutorial, you learn how to verify Firefly Desktop downloads.
+When you download Firefly Desktop, you should verify its authenticity to make sure that you downloaded the correct one from the IOTA Foundation GitHub repository. In this how-to guide, you learn how to verify various Firefly Desktop downloads.
```
Comment (aleksei-korolev) [inline docs/firefly/verify_download.md:15]: either in this guide you will find, or by reading this guide you will learn, or this guide explains.

Comment (aleksei-korolev) [inline docs/firefly/verify_download.md:15]:
```suggestion
When you download Firefly Desktop, you should verify its authenticity to make sure that you have downloaded the correct one from the IOTA Foundation GitHub repository. In this how-to guide, you learn how to verify various Firefly Desktop downloads.
```

Btw "downloaded" is hellowa akward word, is there a synonym?

File: docs/firefly/verify_download.md [hunk 3]
```diff
@@ -8,13 +8,14 @@ keywords:
...
 
 
-To verify the authenticity of Firefly Desktop, you can check its SHA256 hash and its code signature. Instructions for both of these steps differ, depending on your operating system.
+You can verify the authenticity of the Firefly Desktop by checking its SHA256 hash and code signature. The instructions for both of these steps differ, depending on your operating system.
```
Comment (aleksei-korolev) [inline docs/firefly/verify_download.md:18]: Tbh I think you went the wrong direction. The previous version was more lively.

File: docs/firefly/verify_download.md [hunk 4]
```diff
@@ -45,22 +46,22 @@ certUtil -hashfile C:\Users\yourname\Downloads\firefly-desktop-1.0.0.exe SHA256
...
 
-    - DigiCert
-    - DigiCert SHA2 Assured Code Signing CA
-    - IOTA Stiftung
+    - DigiCert.
+    - DigiCert SHA2 Assured Code Signing CA.
+    - IOTA Stiftung.
 
-4. Make sure that the Certificate status reads "This certificate is OK.".
+4. Make sure that the Certificate status reads, "This certificate is OK.".
```
Comment (aleksei-korolev) [inline docs/firefly/verify_download.md:53]: I don't think that this comma belongs here.

File: docs/firefly/verify_download.md [hunk 5]
```diff
@@ -138,7 +139,7 @@ To follow these instructions you need the [Xcode Command Line Tools](https://www
 
 #### Prerequisites
 
-To follow these instructions you need the `sha256sum` package, which is included with most Linux distributions.
+You will need the `sha256sum` package, which is included with most Linux distributions, to complete these instructions.
```
Comment (aleksei-korolev) [inline docs/firefly/verify_download.md:142]: I'd say use simple present and also unwrap the clause into a separate sentence.

File: docs/introduction/path_to_chrysalis.md [hunk 6]
```diff
@@ -24,20 +25,20 @@ The intended outcomes for Chrysalis were:
...
 - **Substantial performance improvements** - With the changes introduced by Chrysalis, we saw a substantial improvement in the scalability and reliability of the IOTA Mainnet.
-- **Improved developer and user experience** - The new protocol features, new libraries, and the new wallet madeIOTA one of the best platforms to build on, removing the friction which developers experience today while allowing solutions built on top of the protocol to provide better user experience.
-- **Accelerated adoption** - Chrysalis made IOTA production-ready by becoming a stable protocol, with a reliable set of developer tools and frameworks that will enabled startups, corporations, and governments to develop and launch products powered by IOTA.
+- **Improved developer and user experience** - The new protocol features, new libraries, and the new wallet made IOTA one of the best platforms to build on, removing the friction which developers experience today while allowing solutions built on top of the protocol to provide a better experience for you.
```
Comment (aleksei-korolev) [inline docs/introduction/path_to_chrysalis.md:28]: > one of the best platforms to build on, removing the friction which developers experience today while allowing solutions built on top of the protocol to provide a better experience for you.

Sorry, but what this actually means, how can it be measured and how can it be verified by the reader?

File: docs/introduction/path_to_chrysalis.md [hunk 7]
```diff
@@ -24,20 +25,20 @@ The intended outcomes for Chrysalis were:
...
+- **Accelerated adoption** - Chrysalis made IOTA production-ready by becoming a stable protocol, with a reliable set of developer tools and frameworks that enabled startups, corporations, and governments to develop and launch products powered by IOTA.
 
 ## The Stages to Chrysalis
 
 ![The stages to Chrysalis](/img/introduction/path_to_chrysalis/01.svg)
 
-The Chrysalis upgrade was a complex undertaking. We coordinated a number of distinct products to ensure a smooth transition for IOTA’s current users and partners. In addition to the core node software, we also needed to update our wallet software, our libraries, and the entire infrastructure.  
+The Chrysalis upgrade was a complex undertaking. We coordinated a number of distinct products to ensure a smooth transition for IOTA’s you and the partners. In addition to the core node software, we also needed to update our wallet software, our libraries, and the entire infrastructure.
```
Comment (aleksei-korolev) [inline docs/introduction/path_to_chrysalis.md:35]: > for IOTA’s you

File: docs/introduction/path_to_chrysalis.md [hunk 8]
```diff
@@ -84,7 +85,7 @@ The Chrysalis changes were specified in the form of RFCs. You can find all the R
...
+The Trinity wallet was a popular IOTA wallet. With Chrysalis, we released a new wallet implementation, [Firefly](https://blog.iota.org/firefly-iota-next-generation-wallet-26bdd4d01510/). The team worked on a complete reevaluation of the wallet architecture, with a completely redesigned experience for you. At its core is a [new wallet library](../libraries/wallet.md) written in Rust. The wallet library is designed to allow other developers to easily implement IOTA wallets within their applications. Another new Rust library utilized by Firefly is Stronghold; Stronghold enables ultra-secure secret handling and storage.
```
Comment (aleksei-korolev) [inline docs/introduction/path_to_chrysalis.md:88]: What's the motivation behind ctrl+fing the "user"? I am all in for making documentation more lively, but I think you are trying too hard with some of it.

File: docs/introduction/what_is_chrysalis.md [hunk 9]
```diff
@@ -14,13 +15,13 @@ The objective of the IOTA Foundation was to optimize the IOTA mainnet before Coo
...
+Why is this process of adopting major protocol improvements relatively unique to IOTA among permissionless Digital Ledger Technologies (DLTs)? The simple answer is the absence of miners. In most permissionless DLTs, the miners’ economic incentives differ from those of regular network users. Changes to throughput and network latencies can disrupt the fee market the miners rely on. This makes them more likely to object to network upgrades as it affects their bottom line.
 
-In IOTA, validators and users are one and the same. There is no conflict of interests between parties with different motivations, meaning there is a much smoother path to network improvements. This is why we are able to incrementally and smoothly upgrade the network before Coordicide.
+In IOTA, you and validators are one and the same. There is no conflict of interests between either with different motivations, meaning there is a much smoother path to network improvements. This is why we are able to incrementally and smoothly upgrade the network before Coordicide.
```
Comment (aleksei-korolev) [inline docs/introduction/what_is_chrysalis.md:24]: same concern.

**Review by aleksei-korolev — CHANGES_REQUESTED** [review]

File: docs/firefly/verify_download.md (hunk 2 above)
Comment (charlesthompson3) [reply docs/firefly/verify_download.md:15]: "Transferred" comes closest but thats not quite good enough, imo.

File: docs/firefly/verify_download.md (hunk 4 above)
Comment (charlesthompson3) [reply docs/firefly/verify_download.md:53]: Quotes preceded by verbs get commas. "Signal verbs". Depends on your grammar style so ymmv.

File: docs/introduction/path_to_chrysalis.md (hunk 6 above)
Comment (charlesthompson3) [reply docs/introduction/path_to_chrysalis.md:28]: I'm not sure I understand the correction/suggestion?

File: docs/introduction/path_to_chrysalis.md (hunk 8 above)
Comment (charlesthompson3) [reply docs/introduction/path_to_chrysalis.md:88]: 🤷

File: docs/firefly/verify_download.md (hunk 5 above)
Comment (charlesthompson3) [reply docs/firefly/verify_download.md:142]: I don't think it needs a separate sentence; its pretty much a "same breath" kind of deal.

File: docs/introduction/path_to_chrysalis.md (hunk 6 above)
Comment (aleksei-korolev) [reply docs/introduction/path_to_chrysalis.md:28]: IOTA being best is subjective, and friction or quality of an experience is abstract.

File: docs/firefly/verify_download.md (hunk 4 above)
Comment (aleksei-korolev) [reply docs/firefly/verify_download.md:53]: Is it actually a quote? :thinking: 

> The plate says "painted".
> The plate says, "painted".
> The plate says: "painted".

@lucas-tortora wdyt?

File: docs/firefly/verify_download.md (hunk 5 above)
Comment (lucas-tortora) [reply docs/firefly/verify_download.md:142]: You could remove the "to complete these instructions" as it's kind of implied.

File: docs/introduction/what_is_chrysalis.md [hunk 10]
```diff
@@ -14,13 +15,13 @@ The objective of the IOTA Foundation was to optimize the IOTA mainnet before Coo
 
 ![IOTA1.0 vs Chrysalis(IOTA1.5) confirmation time](/img/introduction/what_is_chrysalis/00.gif)
 
-A chrysalis is “the form a caterpillar takes before it emerges from its cocoon as a fully-formed moth or butterfly”. In the context of IOTA, Chrysalis was the mainnet’s intermediate stage before Coordicide's completion. The main purpose of Chrysalis is to improve the usability of the previous IOTA mainnet, for users and developers alike.
+A chrysalis is “the form a caterpillar takes before it emerges from its cocoon as a fully-formed moth or butterfly”. For IOTA, Chrysalis was the mainnet’s intermediate stage before Coordicide's completion. The purpose of Chrysalis is to improve the usability of the previous IOTA mainnet, for you and developers.
```
Comment (lucas-tortora) [inline docs/introduction/what_is_chrysalis.md:18]: Isn't the user probably a dev?

File: docs/node_software.md [hunk 11]
```diff
@@ -28,10 +29,10 @@ At this stage, we recommend using Hornet since it has several optional features
 
 ## Node Software
 
-- [hornet](https://wiki.iota.org/hornet/welcome)
-- [bee (branch: chrysalis-pt-2)](https://github.com/iotaledger/bee/tree/chrysalis-pt-2)
+- [hornet](https://wiki.iota.org/hornet/welcome).
```
Comment (lucas-tortora) [inline docs/node_software.md:32]:
```suggestion
## Hornet

- [Official GitHub Repository](https://github.com/gohornet/hornet)
- [Documentation](https://wiki.iota.org/hornet/welcome).
```

File: docs/node_software.md [hunk 12]
```diff
@@ -28,10 +29,10 @@ At this stage, we recommend using Hornet since it has several optional features
...
+- [bee (branch: chrysalis-pt-2)](https://github.com/iotaledger/bee/tree/chrysalis-pt-2).
```
Comment (lucas-tortora) [inline docs/node_software.md:33]:
```suggestion

## Bee

- [Official GitHub Repository](https://github.com/iotaledger/bee/tree/chrysalis-pt-2)
- [Documentation](https://wiki.iota.org/bee/welcome).
```

File: docs/node_software.md [hunk 13]
```diff
@@ -28,10 +29,10 @@ At this stage, we recommend using Hornet since it has several optional features
 
 ## Node Software
```
Comment (lucas-tortora) [inline docs/node_software.md:30]:
```suggestion
```

File: docs/node_software.md (hunk 11 above)
Comment (lucas-tortora) [reply docs/node_software.md:32]: If you want to go down this road, you could also refactor the previous text into each header.

File: docs/firefly/introduction.md [hunk 14]
```diff
@@ -5,6 +5,7 @@ keywords:
 - Firefly
 - DLT
 - wallet
+- reference
 ---
 # Firefly
```
[the remark this answers is not in the document]
Comment (charlesthompson3) [reply docs/firefly/introduction.md:10]: Originally left a review note saying that in the text but axed it, yeah, I'm on it 👌

File: docs/node_software.md (hunk 12 above)
Comment (charlesthompson3) [reply docs/node_software.md:33]: Done 👍

File: docs/introduction/path_to_chrysalis.md (hunk 6 above)
Comment (charlesthompson3) [reply docs/introduction/path_to_chrysalis.md:28]: Hype tweak👍

File: docs/firefly/verify_download.md (hunk 5 above)
Comment (charlesthompson3) [reply docs/firefly/verify_download.md:142]: Was overruled here👌

File: docs/firefly/verify_download.md (hunk 4 above)
Comment (charlesthompson3) [reply docs/firefly/verify_download.md:53]: Gang up here, but removed the comma 😒

File: docs/firefly/verify_download.md (hunk 2 above)
Comment (charlesthompson3) [reply docs/firefly/verify_download.md:15]: Downloaded a-ok👌

**Comment by charlesthompson3** [comment]
@aleksei-korolev @lucas-tortora, incorporated your suggestions.

File: docs/firefly/verify_download.md [hunk 15]
```diff
@@ -1,20 +1,35 @@
...
+
+Firefly is IOTA's new official wallet for Chrysalis. 
+
+:::
+
+**Website**: [https://firefly.iota.org/](https://firefly.iota.org/)
+
+**Code Repository**: [https://github.com/iotaledger/firefly](https://github.com/iotaledger/firefly)
+
 # Verify your Firefly Desktop Download
```
Comment (lucas-tortora) [inline docs/firefly/verify_download.md:27]: Docusuarus only accepts one h1, please  make this a h2 and add one level to subsequent headers

Comment (charlesthompson3) [reply docs/firefly/verify_download.md:27]: 👌

**Review by lucas-tortora — APPROVED** [review]
LGTME!

**Pull request merged**